content/en/docs/concepts/architecture/cloud-controller.md

```diff
@@ -109,7 +109,7 @@ The PersistentVolumeLabels controller moves the cloud-dependent functionality of
 
 ## Plugin mechanism
 
-The cloud controller manager uses Go interfaces to allow implementations from any cloud to be plugged in. Specifically, it uses the CloudProvider Interface defined [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/cloud.go).
+The cloud controller manager uses Go interfaces to allow implementations from any cloud to be plugged in. Specifically, it uses the CloudProvider Interface defined [here](https://github.com/kubernetes/cloud-provider/blob/9b77dc1c384685cb732b3025ed5689dd597a5971/cloud.go#L42-L62).
```

Comment:
Not sure we should ref a commit or some branch. cc @chenopis

Reply:
I ref a commit because of the line numbers I also reference. If we ref a branch the line numbers can easily become invalid.

Reply:
SGTM :) @chenopis WDYT?

Reply:
This makes sense to me. Thanks for the change!